tilgangskontroll-tiltaksansvarlig/src/main/kotlin/no/nav/amt/tiltak/tilgangskontroll_tiltaksansvarlig/metrics/TiltaksansvarligGjennomforingTilgangMetricRepository.kt

```diff
@@ -0,0 +1,25 @@
+package no.nav.amt.tiltak.tilgangskontroll_tiltaksansvarlig.metrics
+
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Component
+
+@Component
+class TiltaksansvarligGjennomforingTilgangMetricRepository(
+	val jdbcTemplate: NamedParameterJdbcTemplate,
+) {
+	fun antallGjennomforingerUtenTilgangerMedMeldinger(): Int {
+		val sql = """
+			SELECT COUNT(DISTINCT g.id) AS antall
+			FROM gjennomforing g
+					 JOIN deltaker d ON d.gjennomforing_id = g.id
+					 JOIN endringsmelding e ON d.id = e.deltaker_id
+			WHERE e.status = 'AKTIV' AND NOT EXISTS (
+					SELECT gjennomforing_id
+					FROM tiltaksansvarlig_gjennomforing_tilgang tt
```

Comment:
Syns det blir litt mer lesbart om man isteden for å bruke forkortelser, kaller det `tilgang`